BBB_cci_mpf/test/test-mpf/base/sw/cci_test_main.cpp

```diff
@@ -251,29 +284,36 @@ int main(int argc, char *argv[])
 
         cout << "#" << endl;
         cout << "#   WRO conflict cycles RR:   " << wro_stats.numConflictCyclesRR;
-        if (cycles != 0)
-        {
-            cout << "  (" << boost::format("%.1f") % (double(wro_stats.numConflictCyclesRR) * 100.0 / cycles) << "% of cycles)";
-        }
-
-        cout << endl << "#   WRO conflict cycles RW:   " << wro_stats.numConflictCyclesRW;
-        if (cycles != 0)
-        {
-            cout << "  (" << boost::format("%.1f") % (double(wro_stats.numConflictCyclesRW) * 100.0 / cycles) << "% of cycles)";
-        }
-
-        cout << endl << "#   WRO conflict cycles WR:   " << wro_stats.numConflictCyclesWR;
-        if (cycles != 0)
+        try  
```

Comment:
What exception can be raised here?

Reply:
Looks like [boost::format()](https://www.boost.org/doc/libs/1_77_0/boost/format/exceptions.hpp) can cause exceptions.